### DISCUSSION TITLE
Export should contain Users, Email-Accounts, Views, etc.

### DISCUSSION THREAD

**Discussion by Bart1909** [jonaswinkler/paperless-ng#770] [Feature Requests]
I had to set up my paperless environment again.
I used the document importer with a previous generated export.
All documents, tags, etc. are imported fine, which is great!

But the users, email accounts, saved views, auth tokens were not included in the export.

It would be great, if this would also be exported, so a restore would be a full restore and do not need any new configuration.

**Comment by jonaswinkler** [comment]
Agreed.

**Reply by Bart1909** [reply]
Thank you very much!
Is there a way to support you? Maybe GitHub Sponsors or Buy me a Coffee?

**Reply by jonaswinkler** [reply]
We've had this discussion before (see #167 , #321). I'd really like to do that, but: Paperless is a collaborative effort. The original project had countless contributors over the years and this fork has also seen contributions from other people. I fear that once I open up donations, other people will probably ask for their share of the donated money.
